appinventor/codelabs/paint-pot/index.html

```diff
@@ -0,0 +1,221 @@
+
+<!doctype html>
+
+<html>
+<head>
+  <meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1.0, user-scalable=yes">
+  <meta name="theme-color" content="#4F7DC9">
+  <meta charset="UTF-8">
+  <title>Paint Pot Codelab</title>
+  <script src="../../bower_components/webcomponentsjs/webcomponents-lite.min.js"></script>
+  <link rel="import" href="../../elements/codelab.html">
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Source+Code+Pro:400|Roboto:400,300,400italic,500,700|Roboto+Mono">
+  <link rel="icon" href="../../elements/favicon.png">
+  <style>
+    body {
+      font-family: "Roboto",sans-serif;
+    }
+  </style>
+
+</head>
+<body unresolved class="fullbleed">
+
+  <google-codelab title="Paint Pot Codelab" feedback-link="https://goo.gl/forms/9zDi1V3arg167lIY2">
+
+      <google-codelab-step label="Overview">
+        <p>In this codelab, you&#39;ll make an app to draw on your phone.</p>
+        <h3 class="checklist">What you&#39;ll learn</h3>
+        <ul class="checklist">
+          <li>Use the <b>Canvas</b> component for drawing.</li>
+          <li>Learn how to use <b>local variables</b> and <b>global variables</b>.</li>
+          <li>Work with different <b>event handlers</b> and types of blocks.</li>
+        </ul>
+
+      </google-codelab-step>
+
+      <google-codelab-step label="Create a new project">
+      <p>Create a new project and name it "PaintPot"</p>
+      <img src="img/new_project.png">
+      <p>In the <b>Properties</b> panel, change the <b>title</b> to "PaintPot"</p>
+      <img src="img/screen_title.png" style="max-width: 400px">
+
+      </google-codelab-step>
+
+      <google-codelab-step label="Add color buttons">
+      <p>Drag three <b>Button</b> components into your workspace. Name the button components "ButtonRed","ButtonBlue", and "ButtonGreen".</p>
```

Comment:
Add space after "ButtonRed"

(Also, you could consider using `<code>` tags)

Reply:
hmm i've been saving the <code> tags for the actual code blocks, but if you want anything typed to be <code> then i can change it!

Reply:
Keeping it in quotes is fine :)